packages/plugin-core/src/abTests.ts

```diff
@@ -0,0 +1,25 @@
+import { ABTest } from "@dendronhq/common-server";
+
+// ^xi5t1r2j51ot
+
+export enum UpgradeToastOrViewTestEnum {
```

Comment:
maybe we should have a convention around these to suffix these enums with "Groups", so `UpgradeToastOrViewTestGroups`, or `UpgradeNotificationTypeTestGroups`

Reply:
+1 to Group Suffix